app/services/trading_strategies.py

```
@@ -4215,6 +4614,304 @@ async def _estimate_daily_volatility(self, symbol: str) -> float:
+                        "target_symbol": symbol,
+                        "hedge_instrument": "BTC" if symbol != "BTC" else "ETH",
+                        "hedge_ratio": min(0.5, concentration / 100),
+                        "rationale": f"Hedge against {symbol} concentration risk",
+                        "urgency": "HIGH" if risk_score > 0.8 else "MEDIUM"
+                    })
+            
+            return opportunities
+        except Exception:
+            return []
+
+    async def _get_exchange_margin_info(self, exchange: str, user_id: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `exchange`

(ARG002)